### ISSUE TITLE
Switch component does not change underlying checkbox "checked" state

### ISSUE THREAD

**Issue by mkarajohn** [Orfium/orfium-ictinus#586]
While it does update the `aria-checked` attribute it does not actually change the `checked` value.

The `checked` value should be preferred over `aria-checked` since this is a native checkbox element.

See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-checked
